pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -879,7 +905,14 @@ func (oe *operationExecutor) generateUnmountDeviceFunc(
 				deviceToDetach.VolumeSpec.Name(),
 				err)
 		}
-
+		refs, err := attachableVolumePlugin.GetDeviceMountRefs(deviceMountPath)
```

Comment:
That's interesting. Could you share an example of this? I'd like to understand what would cause it get in this state. I want to make sure we don't get stuck being able to unmount indefinitely.